### PR TITLE
Adopt FIWARE contribution guidelines

### PR DESCRIPTION
**Pull request by c0c0n3**
This PR makes QL comply with FiWare tech committee's recent decision about contribution guidelines and CLA---see #389. Specifically we have

* updated the code of conduct (Contributor Covenant Code of Conduct 2.0)
* added a contribution guidelines doc as required by FiWare
* implemented the FiWare PR template
* enforced signing of FiWare CLA on PR submission

Notice the signing of CLA uses a GitHub action that [probably won't work][protect] out of the box for us since our `master` branch is protected. Let's see what happens after the merge, if it actually works then all the better, otherwise we'll have to come up with a workaround.

Also I've added a repo secret called `PERSONAL_ACCESS_TOKEN`---which is what the GitHub action looks for

- https://github.com/smartsdk/ngsi-timeseries-api/pull/390/files#r518955135
- https://github.com/smartsdk/ngsi-timeseries-api/settings/secrets/actions

The secret's value is just a placeholder for now. We'll have to generate a personal access token with a scope of `repo`:

- https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token

and then set `PERSONAL_ACCESS_TOKEN` to that token. I've tried generating one using my personal GitHub account but I couldn't manage to make it specific to this repo, i.e. I can only generate a token that gives access to all of my repos, including my private ones!

[protect]: https://github.com/smartsdk/ngsi-timeseries-api/pull/390/files#r518942126

### CODE REVIEW

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        uses: cla-assistant/github-action@v2.0.1-alpha
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN : ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/cla.json'
+          path-to-document: 'https://fiware.github.io/contribution-requirements/individual-cla.pdf' # e.g. a CLA or a DCO document
+          # branch should not be protected
+          branch: 'master'
```
Comment (c0c0n3) [inline .github/workflows/cla.yml:24]: our `master` branch is protected so it looks like we're sitting w/ a problem...

File: .github/workflows/cla.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        # Alpha Release
+        uses: cla-assistant/github-action@v2.0.1-alpha
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN : ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```
Comment (c0c0n3) [inline .github/workflows/cla.yml:19]: Added `PERSONAL_ACCESS_TOKEN` secret to QL repo but the value is bogus, still need to figure out how to generate a valid token which only gives access to the QL repos, not all repos linked to an account.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+- Be kind and thoughtful in your conversations around this project.
+  We all come from different backgrounds and projects, which means we
+  likely have different perspectives on "how open source is done."
+  Try to listen to others rather than convince them that your way is
+  correct.
+- This project is released with a [Contributor Code of Conduct][conduct].
+  By participating in this project, you agree to abide by its terms.
+- If you open a pull request, you must sign the [Individual Contributor
+  License Agreement][cla] by stating in a comment _"I have read the CLA
```
Comment (chicco785) [inline CONTRIBUTING.md:21]: formatting is broken

File: .github/workflows/cla.yml (hunk 1 above)
Comment (chicco785) [reply .github/workflows/cla.yml:24]: then, do we need to be master?

Comment (chicco785) [reply .github/workflows/cla.yml:24]: or can we give special rights to a specific token?

Comment (chicco785) [reply .github/workflows/cla.yml:24]: https://github.com/cla-assistant/github-action if it's only for storing signatures, a separate branch could be a solution 

@jason-fox may have advices

Comment (c0c0n3) [reply .github/workflows/cla.yml:24]: > separate branch could be a solution

perhaps. needs testing

File: CONTRIBUTING.md (hunk 3 above)
Comment (c0c0n3) [reply CONTRIBUTING.md:21]: Uh? GitHub page seems okay? i.e. markdown gets rendered properly and you can follow the links:

- https://github.com/smartsdk/ngsi-timeseries-api/blob/feature/cla/CONTRIBUTING.md

**Review by chicco785 — APPROVED** [review]

**Pull request merged**